app/assets/javascripts/settings-controller.coffee

```diff
@@ -60,35 +62,57 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>                           
+  </div>
 </div>
 
 
 <h1 class="settings-subheader">Works in Progress</h1>
-<a><center> Here you can access all the models that have been edited</center></a>
-  
+<div class="description">Here are all the models you have made changes to in NetLogo Web.</div>
+
 <ul>
   {{#each workInProgressLinks}}
-    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
-    <br>
+    <li><a href="{{url}}">{{modelTitle}}</a> {{storageTag}} <div>{{dateAccessed}} at {{timeAccessed}}</div></li>
   {{/each}}
 </ul>
 
 """
+#Formatting Links for HTML display
 formatLinks = (wipStorage) ->
   result = []
 
   for url, model of wipStorage.inProgress
-    if model.title and model.timeStamp
-      formattedUrl = "/launch#http://" + url.replace(/^url:\/\//, '')
+    if model.title and model.timeStamp and not(url.startsWith('disk') or url.startsWith('new')) and model.version == WIP_INFO_FORMAT_VERSION
+      storageTagOutput = ""
+      storageTag = url.split(':').shift();
+      if storageTag !="url"
+        storageTagOutput = " (tag: " + storageTag + ")"
+        formattedUrl = "/launch?storageTag=#{storageTag}##{window.location.protocol}//" + url.replace(/.*:\/\//, '')
+      else
+        formattedUrl = "/launch##{window.location.protocol}//" + url.replace(/^url:\/\//, '')
+
       result.unshift({
         modelTitle: model.title,
         url: formattedUrl,
-        dataAccessed: new Date(model.timeStamp).toLocaleString()
+        dateAccessed: new Date(model.timeStamp).toLocaleDateString('locale', {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        }),
+        timeAccessed: new Date(model.timeStamp).toLocaleTimeString('locale', {
+          hour: '2-digit',
+          minute: '2-digit',
+          second: '2-digit',
         })
+        storageTag: storageTagOutput
+
+      })
+
+  result.sort ((b, a) ->
+    dateTimeA = new Date("#{a.dateAccessed} #{a.timeAccessed}")
+    dateTimeB = new Date("#{b.dateAccessed} #{b.timeAccessed}")
+    dateTimeA - dateTimeB)
```

Comment:
This sort works, but it seems odd to take the formatted date/time strings and use the `Date()` constructor to parse them back into comparable values.  Can we instead just add the `timeStamp` to the results and use that here?